okio/jvm/jvm.gradle

```diff
@@ -5,9 +5,39 @@ kotlin.targets.matching { it.platformType.name == 'jvm' }.all { target ->
   target.project.sourceCompatibility = JavaVersion.VERSION_1_7
   target.project.targetCompatibility = JavaVersion.VERSION_1_7
 
-  tasks['jar'].configure {
+  // task to generate a OSGi compatible JAR file
+  tasks.register('bundle', rootProject.bndBundleTaskClass) {
+    from sourceSets.main.output
+    destinationDir = file("$buildDir/bundle")
+    bnd = '''
+    Export-Package: okio
+    Automatic-Module-Name: okio
+    Bundle-SymbolicName: com.squareup.okio
+    '''
+  }
+  
+  // extract only the MANIFEST.MF file from the OSGi compatible JAR file
```

Comment:
Does this part do anything meaningful, or is it just because with the old plugin things went sideways here? If this is just ceremony to make me feel better, I think we should remove it.

Reply:
@swankjesse to be honest it's ceremony to make you feel better. I wanted to reduce the surface area the bnd plugins is working on (if you do not trust what the plugin does) therefore my idea was to let the plugin generate the jar file and simply use only the META-INF/MANIFEST.MF file for the jar file generated by the jvmJar task. In practice for the bnd instructions used at the moment the resulting jar file of the bnd plugin will be exactly the same as the resulting jar from the jvmJar task except that the MANIFEST.MF contains the proper OSGi metadata.